.github/workflows/system-tests-against-emulator.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License..
+
+export SPANNER_EMULATOR_HOST=localhost:9010
+export GCLOUD_PROJECT=emulator-test-project
+echo "Running the Cloud Spanner emulator: $SPANNER_EMULATOR_HOST";
+
+# Download the emulator
+EMULATOR_VERSION=0.7.3
+wget https://storage.googleapis.com/cloud-spanner-emulator/releases/${EMULATOR_VERSION}/cloud-spanner-emulator_linux_amd64-${EMULATOR_VERSION}.tar.gz
+tar zxvf cloud-spanner-emulator_linux_amd64-${EMULATOR_VERSION}.tar.gz
+chmod u+x emulator_main
+
+# Start the emulator
+./emulator_main --host_port $SPANNER_EMULATOR_HOST &
```

Comment:
It would be better to kill the emulator after the test. Something like this: 

```bash
# Start the emulator
./emulator_main --host_port $SPANNER_EMULATOR_HOST &

EMULATOR_PID=$!

# Stop the emulator & clean the environment variable
function cleanup() {
    kill -2 $EMULATOR_PID
    unset SPANNER_EMULATOR_HOST
    unset GCLOUD_TESTS_GOLANG_PROJECT_ID
    echo "Cleanup the emulator";
}
trap cleanup EXIT
```

Reply:
Unfortunately, shell scripts run under GH Actions doesn't recognize functions :(

GH Actions thankfully detect the orphaned process and kills it once the tests are complete.

```
Cleaning up orphan processes
Terminate orphan process: pid (2700) (emulator_main)
```

Reply:
Okay. I see.

Reply:
I inlined the function so now it works :)

Interestingly, it still says it's [killing the orphaned process](https://github.com/googleapis/nodejs-spanner/runs/708230157?check_suite_focus=true).